backend/fault.py

```diff
@@ -79,10 +90,18 @@ class AlarmSeverity(DefaultIntEnum):
         if obj.severity == 3 or obj.status is None:
             raise PVInvalidError(self.pv.pvname)
 
+        # "is not None" means it has a value.
+        # self.ok_value = value stated in spreadsheet
```

Comment:
I would replace "=" with "is the" just because my brain read it as pseudocode at first and I got confused lol